modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/ActionsDropdown.tsx

```diff
@@ -227,6 +227,7 @@ function ActionsDropdown({
 								'ml-1': !Liferay.FeatureFlags['LPS-193005'],
 							}
 						)}
+						data-qa-id="itemActionDropdown"
```

Comment:
Could you make sure if this is a duplicate of another data-qa-id? If it is, please double check whether there is possibility of both uses of data-qa-id being on the same page.

Reply:
This is not a duplicate